src/internal.c

```diff
@@ -1879,6 +1879,14 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
     ctx->noPskDheKe = 1;
 #endif
 
+#if defined(WOLFSSL_QT)
```

Comment:
Does this also need `&& !defined(NO_PSK)` to make sure PSK is enabled?

Reply:
You are correct. Should be protected with !defined(NO_PSK)